modules/shovill/main.nf

```diff
@@ -43,7 +43,7 @@ process SHOVILL {
         --force
 
     cat <<-END_VERSIONS > versions.yml
-    ${getProcessName(task.process)}:
+    ${getProcessName(task.process)}
```

Comment:
the colon should be there?

Reply:
Yes, just commited it, I merged it on the browser and deleted it by mistake